docs/.eslintrc.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
```

Comment:
are any of prettier / eslintrc checks run during CI?

Reply:
Not as far as I know, those configs weren't otherwise here. But this too will be moved in to the extracted theme.